ui/tests/helpers/secrets/policy-generator.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+// This is a policy can both mount and list and create oidc keys
+export const adminPolicy = (mountPath: string) => {
```

Comment:
It doesn't look like `mountPath` is used in the return anywhere here or the helper below - should that be added to the policy path somewhere?

Reply:
That's a left over from when I had it. I'll remove it. This policy I suspect will grow and this will come back but for now it's just the ability to mount an engine and read oidc/keys. I'll think about naming some more here too.